meson.build

```diff
@@ -298,7 +302,7 @@ features = {
     'extra_deps': [ pb_comp, protobuf_dep ]
   },
   'tensorrt-support': {
-    'extra_deps': tensorrt_deps
+    'extra_deps': [ tensorrt_deps ]
```

Comment:
Well.. I don't think it works. It's already array of deps.
How about defining empty `nvinfer_dep, ...` instead of `tensorrt_deps`.

Reply:
I'll declared tensorrt_deps as a empty list.

Reply:
Yeap, looks better